mowgli_etl/pipeline/wdc/wdc_heuristic_product_type_classifier.py

```diff
@@ -7,9 +7,17 @@
 
 class WdcHeuristicProductTypeClassifier(WdcProductTypeClassifier):
     def __init__(self):
+        """
+        Set local NLP to optimize execution
+        """
         self.NLP = load("en_core_web_sm")
```

Comment:
This should really be a lazy class variable.

class X:
    __VAR = None

   def __init__(self):
       if self.__VAR is None: self.__VAR = ...
```

should work. May need `self.__class__.__VAR`.